wgengine/router/firewall/README.md

```diff
@@ -0,0 +1,9 @@
+This is a copy of the `tunnel/firewall` package of
```

Comment:
I can see wanting to patch the name package name "firewall" from upstream, but putting it under "wgengine/router" makes it sound like a portable thing available on different OSes.

Maybe `wgengine/router/windows/firewall` or `tempfork/windows-windows/firewall`?

Reply:
Moved under tempfork. I forgot we have that very useful subdir.